bindings/python/src/alert.cpp

```diff
@@ -480,7 +480,8 @@ void bind_alert()
 
     class_<listen_failed_alert, bases<alert>, noncopyable>(
         "listen_failed_alert", no_init)
-        .def_readonly("endpoint", &listen_failed_alert::endpoint)
```

Comment:
you would need to preserve the endpoints in the python binding too, under the deprecated macro